main.go

```diff
@@ -48,7 +48,7 @@ func parse(in []byte) ([]byte, error) {
 		return nil, err
 	}
 
-	data, ok := s["data"].(map[string]string)
+	data, ok := s["data"].(map[interface{}]interface{})
```

Comment:
With changing type to `map[interface{}]interface{}` we can access and convert any data inside secret.
This new version allows you to decode certificates and tokens from secrets